debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/TaskConnectorConfig.java

```diff
@@ -0,0 +1,25 @@
+package io.debezium.connector.sqlserver;
+
+import io.debezium.config.Configuration;
+import io.debezium.config.Field;
+import org.apache.kafka.common.config.ConfigDef;
+
+public class TaskConnectorConfig {
```

Comment:
Let's rename it to `SqlServerTaskConnectorConfig`, just for uniformity with the rest of the class names.